README.md

```diff
@@ -12,28 +12,57 @@ This is a library of various algorithms that are used in competitive programming
 
 
 ### Bellman Ford Algorithm
- [Bellman Ford Algorithm](Bellman_Ford/BellmanFord.cpp)
+[Bellman Ford Algorithm](Bellman_Ford/BellmanFord.cpp)
+ 
+### Circular Queue
+[Circular Queue](Algorithms/Circular_Queue/CircularQueue.cpp)
+
+### Graphs
+[Breadth First Search](Algorithms/Graphs/breadth_first_search.cpp) <br/>
+[Depth First Search](Algorithms/Graphs/DFS_BFS_in_Graphs.cpp) <br/>
+[Detect Cycle in Undirected Graph](Algorithms/Graphs/Detect_Cycle_In_Undirected_Graph.cpp) <br/>
+[Dijkstra](Algorithms/Graphs/Dijkstra's%20Algorithm.cpp) <br/>
+[Kruskal](Algorithms/Graphs/Kruskal's%20Algorithm.cpp)
+
+### Kadane Algorithm
+[Kadane Algorithm](Kadane/kadane.cpp)
 
 ### Last Men Standing
-[Last_Men_Standing.cpp](Algorithms/Last_Men_Standing)
+[Last Men Standing](Algorithms/Last_Men_Standing)
+
+### Prims Algorithm
+[Prims Algorithm](prims_algorithm.cpp)
 
 ### Sample algorithm
-[disjoint_set_union.cpp](Algorithms/disjoint_set_union.cpp) <br />
-[Floyd_cycle_detection.cpp](Algorithms/Linked_List/Floyd_cycle_detection.cpp)
+[disjoint set union](Algorithms/disjoint_set_union.cpp) <br />
+[Floyd cycle detection](Algorithms/Linked_List/Floyd_cycle_detection.cpp)
+
+### Searching Algorithms
+[Binary Search](Algorithms/Searching/binarysearch.cpp) <br/>
+[Jump Search](Algorithms/Searching/jumpsearch.cpp) <br/>
+[Linear Search](Algorithms/Searching/Linear_Search.cpp)
 
 ### Sorting algorithm 
 [Bubble Sort](Algorithms/Sorting/bubble_sort.cpp) <br/>
 [Heap Sort](Algorithms/Sorting/heap_sort.cpp)<br/>
 [Insertion Sort](Algorithms/Sorting/insertion_sort.cpp) <br/>
 [Merge Sort](Algorithms/Sorting/merge_sort.cpp) <br/>
+[Radix Sort](Algorithms/Sorting/radix%20sort.cpp) <br/>
 [Selection Sort](Algorithms/Sorting/selection_sort.cpp)
 
+### Stacks
+[Implementation](Algorithms/Stacks/Stack%20in%20Array.cpp) <br/>
+[Infix to Postfix](Algorithms/Stack/InfixToPostfix.java) <br/>
+[Postfix](Algorithms/Stack/EvaluatePostfixExpression.txt)
+
 ### Strings
-[Substitution_Cipher](https://github.com/NJACKWinterOfCode/nwoc_algorithms/tree/master/Algorithms/Substitution_Cipher)
+[Substitution Cipher](https://github.com/NJACKWinterOfCode/nwoc_algorithms/tree/master/Algorithms/Substitution_Cipher)
```

Comment:
No change dome over here

Reply:
What are the changes to be done?

Reply:
No i mean the commit does not contain change